trackintel/io/file.py

```diff
@@ -485,13 +497,16 @@ def write_trips_csv(trips, filename, *args, **kwargs):
 
     Parameters
     ----------
-    trips : DataFrame (as trackintel trips)
+    trips : (Geo)DataFrame (as trackintel trips)
         The trips to store to the CSV file.
 
     filename : str
         The file to write to.
     """
     df = trips.copy()
+    if isinstance(df, GeoDataFrame):
+        geom_col_name = df.geometry.name
+        df[geom_col_name] = df[geom_col_name].apply(wkt.dumps)
```

Comment:
I am now using `isinstance` in all cases to check whether the geometry exists or not. We have to be careful with GeoDataFrames that do not contain a geometry in further PRs